src/target/source/codegen_cuda.cc

```diff
@@ -723,6 +724,41 @@ void CodeGenCUDA::VisitExpr_(const CallNode* op, std::ostream& os) {
       this->PrintExpr(op->args[i * 2 + 1], os);
       os << "]" << ((i < 3) ? ", " : ")");
     }
+  } else if ((op->op.same_as(builtin::call_extern()) ||
+              op->op.same_as(builtin::call_pure_extern())) &&
+             Downcast<StringImm>(op->args[0])->value == "ptx_mma") {
```

Comment:
It would be better to explicitly define it as `builtin::ptx_mma` op instead of using call extern. This would make other rewrite passes to be aware of this op and handle it explicitly if needed